QModManager/HarmonyPatches/EnableConsoleSetting.cs

```diff
@@ -4,16 +4,19 @@
     using QModManager.Utility;
     using UnityEngine;
 
-    [HarmonyPatch(typeof(DevConsole), nameof(DevConsole.Awake))]
-    internal static class DevConsole_Awake_Patch
+    [HarmonyPatch(typeof(DevConsole), nameof(DevConsole.Update))]
```

Comment:
Is there a reason we need to patch `Update` now instead of `Awake` or `Start`?  
We don't need this patch to run multiple times.

If we must patch the `Update` method, we should at least try to unpatch after its first run.

Reply:
Unfortunately because of the patch time change we no longer start patching things before the awake or start are called. Not 100% sure why I didnt unpatch it though

Reply:
Actually removed the update as I moved the patch time back before that start so it will work again.